third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -1,3 +1,4 @@
+<% autogen_exception -%>
 package google
 
```

Comment:
There are a couple places in this file and the test file where you'll need to change `clientCompute` to `clientComputeBeta`. Same with changing `computeOperationWait` to `computeSharedOperationWait`.